### PR TITLE
Changes for Android

### PR DESCRIPTION
**Pull request by cgrigis**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -9,7 +9,7 @@ edition = "2018"
 
 [lib]
 name = "core"
-crate-type = ["staticlib"]
+crate-type = ["cdylib", "staticlib"]
```
Comment (cgrigis) [inline Cargo.toml:12]: The Android JNI requires a dynamic library

File: src/arti.rs [hunk 2]
```diff
@@ -33,8 +33,6 @@ include_str!("./authorities.toml"),
 /// This connection sends a GET request over TLS to the domain.
 /// It returns the result from the request, or an error.
 pub fn tls_get(domain: &str, cache_dir: Option<&str>) -> Result<String> {
-    simple_logging::log_to_stderr(LevelFilter::Debug);
```
Comment (cgrigis) [inline src/arti.rs:36]: This seems to conflict with the Android logging that is setup before.

File: Cargo.toml (hunk 1 above)
Comment (ineiti) [reply Cargo.toml:12]: This will give a warning with the iOS build, but we'll ignore it for the moment.

**Review by ineiti — APPROVED** [review]
I'll merge it, even with the build fail, and hope that #11 will take care of it...

**Pull request merged**